sessionctx/stmtctx/stmtctx_test.go

```diff
@@ -48,3 +48,31 @@ func TestCopTasksDetails(t *testing.T) {
 		t.Fatal(c)
 	}
 }
+
+func TestStatementContextPushDownFLags(t *testing.T) {
```

Comment:
It's better to use `github.com/pingcap/check` to run unit tests (make unit tests parallelable).